backbone/src/test/java/org/researchstack/backbone/task/factory/TappingTaskTest.java

```diff
@@ -31,8 +27,8 @@
 
 public class TappingTaskTest {
 
-    private Context mockContext;
-    private Resources mockResources;
+    @Mock private Context mockContext;
```

Comment:
Ohh that makes sense! I thought it was more of a code comment than an injection annotation, but I will do this from now on.